src/Storage/Storage.Management/Storage.Management.csproj

```diff
@@ -15,9 +15,9 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Storage" Version="13.2.0" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.1.0" />
-    <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.0" />
-    <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="11.1.0" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="10.0.3-Release-jenkins-dotnet-nuget-166" />
+    <PackageReference Include="Microsoft.Azure.Storage.File" Version="10.0.3-Release-jenkins-dotnet-nuget-166" />
+    <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="10.0.3-Release-jenkins-dotnet-nuget-166" />
```

Comment:
This is a private version, won't be in final code.
Send the code just for review early since the code change is big.

When get the signed the XSCL for this, the version won't be downgrade.